libs/cli/langchain_cli/package_template/README.md

```diff
@@ -1 +1,35 @@
 # __package_name_last__
+
+TODO: What does this package do
+
+## Environment Setup
+
+TODO: What environment variables need to be set (if any)
+
+## Usage
+
+To use this package, you should first have the LangChain CLI installed:
+
+```shell
+pip install -U "langchain-cli[serve]"
+```
+
+To create a new LangChain project and install this as the only package, you can do:
+
+```shell
+langchain serve new my-app
+cd my-app
+langchain serve add __package_name_last__
+```
+
+If you want to add this to an existing project, you can just run:
+
+```shell
+langchain serve add __package_name_last__
+```
+
+If you are inside this directory, then you can spin up a LangServe instance directly by:
+
+```shell
+langchain hub start
```

Comment:
We should add context on how to connect -

This will start the FastAPI app with a server is running locally. E.g., 
```
Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
```

We can see all templates:
```
http://127.0.0.1:8000/docs#
```

We can access the playground:
```
http://127.0.0.1:8000/__package_name_last__/playground
```

We can access the template from code:
```python
from langserve.client import RemoteRunnable
runnable = RemoteRunnable("http://0.0.0.0:8001/__package_name_last__")
```